frontend/src/components/app/Modal.js

```diff
@@ -214,6 +215,7 @@ class Modal extends Component {
             processType: windowId,
             viewId,
             parentId: parentWindowId,
```

Comment:
I'm a bit confused. We are expecting in here parentWindowId and this is not passed anymore. This may lead to future logic bugs.

Reply:
Yeah, a leftover since we don't use this anywhere